torax/sources/default_sources.py

```diff
@@ -46,6 +48,8 @@ def get_default_runtime_params(
       return external_current_source.RuntimeParams()
     case 'nbi_particle_source':
       return electron_density_sources.NBIParticleRuntimeParams()
+    case 'echcd':
```

Comment:
hey Theo :) We've just made a change to simplify adding sources which involves removing default sources.

So you could remove these changes (or accept the file as deleted in a merge).

The only change that is required to hook up a source and make it discoverable to torax now is to register a source. Docs for this incoming soon but for letting you progress this CL will add below too. 

To register a source you can use the `register_source` method https://github.com/google-deepmind/torax/blob/main/torax/sources/register_source.py#L63.

We do this [here](https://github.com/google-deepmind/torax/blob/main/torax/sources/register_source.py#L164) for the commonly used TORAX sources but in general the idea is a user could use this method to register a source in any module (provided that module is imported by Python).

Registering a source requires you to pass in a reference to the `Source` you create and its `RuntimeParams` object and register that source under a `source_name`. The `source_name` you register under will be the thing that would identifying your source and be the key for your source if you wish to use it in the python experiment config e.g. for [here](https://github.com/google-deepmind/torax/blob/main/torax/examples/iterhybrid_rampup.py#L115) for the source registered and identified as `gas_puff_source`.

Hope this helps and sorry again or you needing to make additional changes, hopefully this will be a simpler pattern overall.

Let us know if that requires clarification or any further help!

Reply:
No problem! New pattern looks great :) I think I've made the appropriate changes